### PR TITLE
Added ErrorBoundary to Map and context wrapped components

### PR DESCRIPTION
**Pull request by gabe647**
Resolves: attach_gh_issue_here

### PR Safety Checklist:

 - [ ] Added the task to the appropriate release doc under **Enhancements** or **Bug Fixes**
 - [ ] Bump `package.json` & `package-lock.json` version numbers to appropriate release
 - [ ] (optional) All external API changes have been documented
 - [ ] (optional) Build docs: `npm run docs`

### CODE REVIEW

File: src/LayerStyler/_Selector/index.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ class Selector extends Component {
     }
 
     return (
-      <Container style={styles} {...this.props}>
+      <Container style={styles} selected={selected} disabled={disabled}>
```
Comment (gabe647) [inline src/LayerStyler/_Selector/index.js:105]: This fixes a React warning

File: src/Project/utils.js [hunk 2]
```diff
@@ -6,8 +6,8 @@ import { loadBasemapLayer } from 'Basemaps'
 import { transform } from 'ol/proj'
 import { centerAndZoom } from 'Map'
 import ugh from 'ugh'
-import { version } from '../../package.json'
-
+import packageJson from '../../package.json'
+const { version } = packageJson
```
Comment (gabe647) [inline src/Project/utils.js:10]: Fixes a node warning

File: src/locales/en.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
-import { version } from '../../package.json'
+import packageJson from '../../package.json'
+const { version } = packageJson
```
Comment (gabe647) [inline src/locales/en.js:2]: Fixes node warning

**Comment by stazrad** [comment]
LGTM just bump the version

**Review by stazrad — DISMISSED** [review]

**Review by akuma1 — APPROVED** [review]

**Pull request merged**